README.md

```diff
@@ -224,7 +224,8 @@ Supported Options Reference
 
 * `Unattended-Upgrade::Remove-New-Unused-Dependencies` - boolean (default:True)
 
- Remove any new unused dependencies after the upgrade has finished.
+ Remove any new unused dependencies after the upgrade has finished. That is
```

Comment:
I think the original sentence can also benefit from rephrasing, like: "Remove packages which became unused during the upgrade".